sentry/src/main/java/io/sentry/ScopesAdapter.java

```diff
@@ -190,6 +195,32 @@ public void flush(long timeoutMillis) {
     return Sentry.getCurrentScopes().clone();
   }
 
+  @Override
+  public @NotNull IScopes forkedScopes(@NotNull String creator) {
+    return Sentry.forkedScopes(creator);
+  }
+
+  @Override
+  public @NotNull IScopes forkedCurrentScope(@NotNull String creator) {
```

Comment:
`l`: I'm wondering if we should have method overloads here instead. E.g. `.forkedScope(ScopeType type)`, Same could be done for the getters.